api/api.go

```diff
@@ -82,19 +86,32 @@ func NewSearchAPI(router *mux.Router, dpESClient DpElasticSearcher, deprecatedES
 // RegisterGetSearch registers the handler for GET /search endpoint
 // with the provided validator and query builder
 // as well as the API's elasticsearch client and response transformer
-func (a *SearchAPI) RegisterGetSearch(validator QueryParamValidator, builder QueryBuilder, transformer ResponseTransformer) *SearchAPI {
+func (a *SearchAPI) RegisterGetSearch(validator QueryParamValidator, builder QueryBuilder, settingsNLP config.NLP, cli *nlp.Client, transformer ResponseTransformer) *SearchAPI {
 	a.Router.HandleFunc(
 		"/search",
 		SearchHandlerFunc(
 			validator,
 			builder,
+			settingsNLP,
+			cli,
 			a.dpESClient,
 			transformer,
 		),
 	).Methods(http.MethodGet)
 	return a
 }
 
+// RegisterGetSearch registers the handler for GET /nlp/search endpoint
```

Comment:
RegisterGetNlpSearch? but also the is the endpoint intended to group `nlp` related resources under `/nlp/` or is this a single endpoint, in which case, something like `search-nlp`  might be more appropriate

Reply:
Well the idea was to group the endpoints, but considering that there's only one, maybe I should've renamed it..
Will do now